<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerLongitudinalPETCTReportSelectionWidget</class>
 <widget class="QWidget" name="qSlicerLongitudinalPETCTReportSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="LabelSelectReport">
       <property name="text">
        <string>Select Report:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxReport" native="true">
         <property name="toolTip">
          <string>Select a Report in order to perform PET/CT analysis</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonModuleSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Default settings for this module.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
           <normaloff>:/Icons/settings_small.png</normaloff>:/Icons/settings_small.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabelInfo">
         <property name="styleSheet">
          <string notr="true">QLabel QToolTip {background-color: #858585; color: #FFF;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qSlicerLongitudinalPETCTModule.qrc">:/Icons/help.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="FrameInformation">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="LabelName">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Patient Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="LabelDoB">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Date of Birth:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LabelSex">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Sex:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="LabelNameInfo">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="LabelDoBInfo">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="LabelSexInfo">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="LabelSize">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="LabelSizeInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkExpandButton" name="ExpandButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Show/hide expandable patient information panel</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="mirrorOnExpand" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkExpandButton</class>
   <extends>QToolButton</extends>
   <header>ctkExpandButton.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerLongitudinalPETCTModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>FrameInformation</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
